heidegger_index/templates/work_detail.html

```diff
@@ -6,7 +6,7 @@
 {% include '_export_button.html' with type="work" %}
 {% include '_home_button.html' %}
 
-{% include 'components/h1.html' with text=work.csl_json.title class="clear-right italic" only %}
+{% include 'components/h1.html' with text=work.csl_json.title class="clear-right italic" complete=work.complete only %}
```

Comment:
Liever niet aan `h1.html` doorgeven, die was generiek bedoeld. Strikt gezien is een `<h1>` ook alleen bedoeld voor tekst. Je kunt dit doen:

```html
<div class="flex">
  {% include 'components/h1.html' with text=work.csl_json.title class="clear-right italic" only %}
  {% if complete %}{% include '../_complete_icon.html' %}{% endif %}
</div>
```
  
```